gorm/utilities_test.go

```diff
@@ -0,0 +1,43 @@
+package gorm
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+type Human struct {
+	Name  string
+	Age   uint32 `gorm:"column:years"`
+	Child *Child
+}
+
+func (p Human) TableName() string {
+	return "db_humans"
+}
+
+type Child struct {
+	Name string
+}
+
+func TestHandleFieldPath(t *testing.T) {
+	tests := []struct {
+		fieldPath []string
+		dbName    string
+		err       bool
+	}{
+		{[]string{"name"}, "db_humans.name", false},
+		{[]string{"age"}, "db_humans.years", false},
```

Comment:
Having `table_name.column` works for standard queries without an `AS` or join? I suppose that makes sense, but I don't know that I've seen it before.

Reply:
If this works in Postgres, won't it be any problems with other DBs like MySQL.

Reply:
For me It seems very natural to work. Table name is not required even in case of join and `AS` if there is no collision between column names across tables. I tested it on postgres and sqlite - works fine. And remember doing similar things with Oracle DB.